plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TransactionLogTail.java

```diff
@@ -94,6 +94,45 @@ public static TransactionLogTail loadNewTail(
         return new TransactionLogTail(entriesBuilder.build(), version);
     }
 
+    // Load a section of the Transaction Log JSON entries. Optionally from a given end version (inclusive) through an start version (inclusive)
+    public static TransactionLogTail loadNewTailBackward(
```

Comment:
Any specific reason for reading backward, because we try to reverse them later in `getPageSource`

Reply:
Yes, The transaction log that we are going to read using `loadNewTailBackward` will start with the last checkpoint file version and go till 0. Even when the log retention interval is exceeded and some of the initial transaction logs (0, 1, ...) are deleted we will able to get the history of the table.